updateStableIds/pom.xml

```diff
@@ -0,0 +1,94 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>org.reactome.release</groupId>
+  <artifactId>updateStableIds</artifactId>
+  <version>0.0.1-SNAPSHOT</version>
+  <packaging>jar</packaging>
+
+  <name>updateStableIds</name>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+  </properties>
+
+  <dependencies>
+  	<dependency>
+	  <groupId>org.reactome.server</groupId>
+   	  <artifactId>reactome-core</artifactId>
+	  <version>1.1.2</version>
+	</dependency>
+	  <dependency>
+	  <groupId>org.reactome.base</groupId>
+	  <artifactId>reactome-base</artifactId>
+	  <version>1.2.15</version>
+	</dependency>
+	<dependency>
+	  <groupId>org.reactome.release</groupId>
+   	  <artifactId>release-common-lib</artifactId>
+	  <version>1.0.2</version>
+	</dependency>
+	<dependency>
+	  <groupId>org.apache.logging.log4j</groupId>
+   	  <artifactId>log4j-core</artifactId>
+	  <version>2.11.0</version>
+	</dependency>
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <version>3.8.1</version>
+      <scope>test</scope>
+    </dependency>
+	  <dependency>
+		  <groupId>junit</groupId>
+		  <artifactId>junit</artifactId>
+		  <version>4.12</version>
+		  <scope>test</scope>
+	  </dependency>
+	  <dependency>
+		  <groupId>org.powermock</groupId>
+		  <artifactId>powermock-api-mockito</artifactId>
+		  <version>1.6.5</version>
```

Comment:
You can move "1.6.5" to a property. Add `<powermockito.version>1.6.5</powermockito.version>` to the `<properties>...</properties>` list, and then you can refer to it on line 52 and 58 as `${powermockito.version}`. Not necessary, but makes version management easier. It's more of an issue when you need to refer to the same version number string for a large list of dependencies.

Reply:
(not necessary for this release, just thought you'd like to know that it's an option)